examples/shader/Shader.cpp

```diff
@@ -21,46 +21,58 @@ std::random_device rd;
 std::mt19937       rng(rd());
 } // namespace
 
+
+////////////////////////////////////////////////////////////
+// Base class for effects
+////////////////////////////////////////////////////////////
+struct Effect : public sf::Drawable
```

Comment:
```suggestion
struct Effect : sf::Drawable
```
Did a quick audit of the codebase and it looks like in all circumstances except 1 (EnumArray) we omit `public` when a struct inherits from another type.